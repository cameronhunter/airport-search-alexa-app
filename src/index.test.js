import Skill from './index';
import Request from 'alexa-request';

test('LaunchRequest', () => {
  const event = Request.launchRequest().build();

  return expect(Skill(event)).resolves.toMatchSnapshot();
});

test('Hello intent', () => {
  const event = Request.intent('hello', { name: 'world' }).build();

  return expect(Skill(event)).resolves.toMatchSnapshot();
});
