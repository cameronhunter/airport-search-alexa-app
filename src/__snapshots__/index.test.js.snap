// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello intent 1`] = `
Object {
  "response": Object {
    "card": Object {
      "content": "Hello world",
      "title": "Alexa App",
      "type": "Simple",
    },
    "outputSpeech": Object {
      "text": "Hello world",
      "type": "PlainText",
    },
    "shouldEndSession": true,
  },
  "version": "1.0",
}
`;

exports[`LaunchRequest 1`] = `
Object {
  "response": Object {
    "outputSpeech": Object {
      "text": "Alexa app launched!",
      "type": "PlainText",
    },
    "shouldEndSession": true,
  },
  "version": "1.0",
}
`;
